src/twisted/web/test/test_http.py

```diff
@@ -3213,6 +3213,35 @@ def test_expect100ContinueHeader(self):
 
 
 
+class ConnectRequestTests(unittest.TestCase, ResponseTestMixin):
+    """
+    Test correct RFC 7231 handling for CONNECT requests. Responses to these
+    tests must not contain Transfer-Encoding or Content-Length headers
+    """
+
+    def test_headers(self):
+        transport = StringTransport()
```

Comment:
Please add docstring here to eliminate error here: https://travis-ci.org/twisted/twisted/jobs/284299844.
When writing docstring, follow this guideline: https://jml.io/pages/test-docstrings.html
and don't put *Test * in the description.